libfwupdplugin/fu-kernel.c

```diff
@@ -305,3 +305,76 @@ fu_kernel_get_cmdline(GError **error)
 	return NULL;
 #endif
 }
+
+gboolean
+fu_kernel_is_cmdline_mutable(void)
+{
+	g_autofree gchar *grubby_path = fu_path_find_program("grubby", NULL);
+	return grubby_path != NULL;
+}
+
+static gboolean
+fu_kernel_set_commandline(const gchar *arg, gboolean value, GError **error)
```

Comment:
As a nit I think you should do:
```suggestion
fu_kernel_set_commandline(const gchar *arg, gboolean enable, GError **error)
```

Reply:
Fixed, thanks.